tests/golos.charge_tests.cpp

```diff
@@ -127,4 +130,26 @@ BOOST_FIXTURE_TEST_CASE(basic_tests, golos_charge_tester) try {
     
 } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(set_params, golos_charge_tester) try {
+    BOOST_TEST_MESSAGE("Test charge parameters");
+    
+    produce_block();
+    
+    BOOST_TEST_MESSAGE("--- check that global params not exist");
+    BOOST_TEST_CHECK(charge.get_params().is_null());
+
+    produce_block();
+
+    auto params = "[" + charge.get_str_vesting_acc(_code) + "]";
+    BOOST_CHECK_EQUAL(err.no_vesting_acc, charge.set_params(params));
```

Comment:
there is need to delay account creation, you can check this by passing any non-existing account